visualization/app/codeCharta/util/envDetector.ts

```diff
@@ -1,5 +1,5 @@
 export function isStandalone(): boolean {
-	return process.env.STANDALONE === undefined ? false : JSON.parse(process.env.STANDALONE)
+	return false
```

Comment:
Check, if this doesn't matter.

Reply:
Do we need the `isStandalone` check at all?